build_tools/python/e2e_test_framework/definitions/common_definitions.py

```diff
@@ -96,6 +96,10 @@ class ModelSourceType(Enum):
   """Type of model source."""
   # Exported Linalg MLIR file.
   EXPORTED_LINALG_MLIR = "exported_linalg_mlir"
+  # Exported MHLO file.
+  EXPORTED_MHLO_MLIR = "exported_mhlo_mlir"
+  # Exported StableHLO file.
+  EXPORTED_STABLEHLO_MLIR = "exported_stablehlo_mlir"
```

Comment:
If the eventual goal is to support stablehlo, I'll suggest adding this later when we are ready to use stablehlo. And at that point we directly modify mhlo import config to be stablehlo import config to avoid new import config id.